utils/ort/src/main/kotlin/OrtAuthenticator.kt

```diff
@@ -70,7 +70,14 @@ class OrtAuthenticator(private val original: Authenticator? = null) : Authentica
 
     // First look if the credentials are already present in the URL, then search for (potentially machine-specific)
     // credentials in a netrc-style file, and finally look for generic credentials passed as environment variables.
-    private val delegateAuthenticators = listOf(UserInfoAuthenticator(), NetRcAuthenticator(), EnvVarAuthenticator())
+    // If all this fails, fall back to the original authenticator if it is defined. This allows extending ORT's
```

Comment:
"fails" maybe is a bit of a hard expression, how about "If none of these can provide credentials, ...".

Reply:
"if it is defined." -> ", if any."

Reply:
Last sentence: "This allows tools that use ORT programmatically to provide a custom authenticator."

Reply:
Reworded as proposed.